tests/test.cpp

```diff
@@ -227,7 +227,7 @@ flatbuffers::DetachedBuffer CreateFlatBufferTest(std::string &buffer) {
 
   FinishMonsterBuffer(builder, mloc);
 
-// clang-format off
+  // clang-format off
```

Comment:
unrelated changes?

Reply:
I formatted the whole file, and these were wrong.

I really wish clang-format comments could be a macro instead of comment.